src/analyze/BuildSystem.re

```diff
@@ -264,10 +264,24 @@ let isRunningInEsyNamedSandbox = () => {
 };
 
 let getExecutableInEsyPath = (exeName, ~pwd) => {
-  if (isRunningInEsyNamedSandbox()) {
+  let ret = if (isRunningInEsyNamedSandbox()) {
     getLine("which " ++ exeName, ~pwd)
   } else {
     getLine("esy which " ++ exeName, ~pwd)
+  };
+  if (Sys.win32) {
+    switch ret {
+      | RResult.Ok(ret) =>
+        let ret = if (isRunningInEsyNamedSandbox()) {
+          getLine("cygpath -w " ++ ret, ~pwd)
```

Comment:
this is only if you're running under cygwin, right? Which I don't think is necessarily the case?

Reply:
We are on windows `if (Sys.win32)` and esy on windows have cygwin with esy-bash: so I assume it is necessarily the case. Maybe ping a maintener of esy :shrug:

Reply:
I can add a try with.